src/jit/compiler.h

```diff
@@ -6625,8 +6626,14 @@ class Compiler
     {
         return codeGen->doDoubleAlign();
     }
-    DWORD getCanDoubleAlign(); // Defined & used only by RegAlloc
-#endif                         // DOUBLE_ALIGN
+    DWORD getCanDoubleAlign();
+    bool shouldDoubleAlign(unsigned refCntStk,
+                           unsigned refCntReg,
+                           unsigned refCntWtdReg,
+                           unsigned refCntStkParam,
+                           unsigned refCntWtdStkDbl);
```

Comment:
I don't think it is under #ifdef DEBUG, unless I am missing something.

---
In reply to: [81882503](https://github.com/dotnet/coreclr/pull/7486#discussion_r81882503) [](ancestors = 81882503)